README.md

```diff
@@ -0,0 +1,47 @@
+# roffcpp
+
+C++ library for parsing Roxar Open File Format (ROFF) (binary and ascii) files.
+
```

Comment:
This library is inspired by https://github.com/equinor/xtgeo and some test data is taken from https://github.com/equinor/xtgeo-testdata

Reply:
Fixed.